packages/tomls/src/omnibus/reya_network.toml

```diff
@@ -116,7 +116,7 @@ ae_liquidator1 = "0x89520d105a125CC6165c6685de262c42113Df9c0"
 [var.co_execution_bots]
 co_execution_bot1 = "0x0d171dFaab3440c0C88F3a07d8F3e9ffE56C609a"
 
-# TODO: confirm that this is the admin for mainnet
+# note, global pausers are also admins for this flag
```

Comment:
I would suggest defining admin2, admin3, admin4, and admin5 as "<%= settings.pauser1 %>", ""<%= settings.pauser2 %>" etc

Reply:
(for better visibility)

Reply:
ok